pylanetary/navigation/core.py

```diff
@@ -148,33 +148,43 @@ def emission_angle(ob_lat, surf_n):
     
     Parameters
     ----------
+    ob_lat : float or np.array, required. 
+        [deg] sub-observer latitude
+    surf_n : np.array, required.
+        [x,y,z] surface normal vector at each ob_lat. must have vector length 1
     
     Returns
     -------
-    
+    mu : [-] cosine of emission angle
     '''
     ob = np.asarray([np.cos(np.radians(ob_lat)),0,np.sin(np.radians(ob_lat))])
```

Comment:
How does planetographic vs planetocentric affect this. If your input latitude is in graphic it should work, I think, but not for centric.

Reply:
I added a line that normalizes it, and removed the phrase requiring it be normalized from the docstring